packages/ddp-client/livedata_connection.js

```diff
@@ -38,7 +38,11 @@ var Connection = function (url, options) {
     reloadWithOutstanding: false,
     supportedDDPVersions: DDPCommon.SUPPORTED_DDP_VERSIONS,
     retry: true,
-    respondToPings: true
+    respondToPings: true,
+    // When updates are coming within this ms interval, batch them together.
+    bufferedWritesInterval: 5,
+    // Flush buffers immediately if writes are happening continuously for more than this many ms.
+    bufferedWritesMaxAge: 500
```

Comment:
No no. Do not change this. The point here is that what is really important to users (I believe) is framerate. Not size of data. You want to have predictable framerate.

To understand what is happening here is that in most cases there is only 5 ms delay, not 500 ms delay. So if changes are coming sporadically, then you have a 5 ms delay while buffering is waiting if there is no other change. If there is no other change, then change go through immediately. If there are changes for more than 500 ms coming without any 5 ms pause between any of those change, then after 500 ms of those sustained changes we flush them. I think this is also good. Because if you have 500 ms of sustained changes coming through then the whole point of this pull request was that we are not passing those changes through because they hog the whole rendering of those changes.

I think those defaults are pretty reasonable. 5 ms delay for rare changes, and 500 ms flushing of big changes which will probably change a lot of DOM anyway. But I agree that there should be a way to change those settings for the default connection. But what would be good API for this? Simple some `Meteor.defaultConnectionSettings` which you can set/override in your app and is then used when creating a default connection?